libs/smart-contracts/src/index.ts

```diff
@@ -1,2 +1,5 @@
 export * from './contracts';
 export * from './utils';
+
+export { default as ERC20_ABI } from './abis/erc20_abi.json';
```

Comment:
I think we can un export these now that we we use the class wrappers for each contract